textrepo-app/src/main/java/nl/knaw/huc/service/task/indexer/JdbiIndexFileTaskBuilder.java

```diff
@@ -21,8 +21,10 @@
 import java.util.Optional;
 import java.util.function.Supplier;
 
+import static com.google.common.primitives.Shorts.asList;
 import static java.lang.String.format;
```

Comment:
Obscure externe dependency. Kan dit ook met Arrays.asList() of Collections.singletonList() (die laatste is immutable) ?

Reply:
7b2bd48